src/js/main-window.js

```diff
@@ -6,9 +6,13 @@ var Accel = require('ui/accel');
 var Utils = require('utils');
 var Menu = require('menu');
 var Elements = require('elements');
+var Feature = require('platform/feature');
+var Settings = require('settings');
+
+Accel.init();
 
 var mainWindow = new UI.Window({
-  fullscreen: true, 
```

Comment:
I found some documentation on this: https://github.com/pebble/pebblejs#window-statusdef